libraries/client/client_webhook.py

```diff
@@ -187,6 +192,12 @@ def process_webhook(self):
             source_url = self.build_multipart_source(file_key, book)
             identifier, job = self.send_job_request_to_tx_manager(commit_id, source_url, rc, repo_name, repo_owner,
                                                                   count=book_count, part=i, book=book)
+
+            # Send lint request to tx-manager
+            lint_results = self.send_lint_request_to_tx_manager(job, source_url)
```

Comment:
I think what I was needing to see was how you merge each build_log for the multipart stuff.... because I am saving the warnings to the job table....in the current code on Github tx-manager saves the job with the linter warnings, but I don't like that..so am going to have the client actually update the job table.